rts/lra/lra-test/basic/src/test/java/io/narayana/lra/arquillian/NonRootLRAParticipantIT.java

```diff
@@ -0,0 +1,56 @@
+package io.narayana.lra.arquillian;
```

Comment:
Hi @xstefank we normally put a copyright notice on even the tests as per https://github.com/jbosstm/narayana/blob/master/rts/lra/lra-test/tck/src/test/java/io/narayana/lra/tck/arquillian/ArquillianLRATckExtension.java#L1 (just need to change the date I think) - same comment for the rest of the added files

Reply:
ah, I always forget, sorry about that

Reply:
No problemo

Reply:
@tomjenkinson thanks, notices added

Reply:
Perfect - thanks @xstefank !